src/components/inputs/TextField.js

```diff
@@ -643,7 +644,8 @@ function createStyles({placeholderTextColor, centered, multiline}) {
       backgroundColor: 'transparent'
     },
     placeholder: {
-      color: placeholderTextColor
+      color: placeholderTextColor,
+      writingDirection: Constants.isRTL ? 'rtl' : 'auto'
```

Comment:
Here 'auto' should be enough, no? It should recognize an RTL layout and flip it automatically.

Reply:
Also worth checking in Android, as I understand this is an iOS prop

Reply:
Android works fine, iOS needs it. 
when using `forceRTL` that's what actually makes it work.